src/features/Library/Applet/Applet.tsx

```diff
@@ -0,0 +1,140 @@
+import { useNavigate } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { Box, Button } from '@mui/material';
+
+import {
+  StyledBodyLarge,
+  StyledBodyMedium,
+  StyledHeadlineLarge,
+  StyledLabelBoldLarge,
+  StyledTitleBoldMedium,
+  StyledTitleMedium,
+} from 'styles/styledComponents';
+import theme from 'styles/theme';
+import { AppletImage } from 'features/Library/AppletImage';
+import { Svg } from 'components';
+import { page } from 'resources';
+import { variables } from 'styles/variables';
+
+import {
+  StyledAppletContainer,
+  StyledAppletName,
+  StyledAppletKeywordsContainer,
+  StyledAppletKeyword,
+  StyledButtonsContainer,
+} from './Applet.styles';
+import { AppletProps, AppletUiType } from './Applet.types';
+import { Activities } from './Activities';
+
+export const Applet = ({
+  applet: { appletId, name, image = '', version = '', description, keywords },
+  uiType = AppletUiType.List,
+}: AppletProps) => {
+  const { t } = useTranslation('app');
+  const navigate = useNavigate();
+
+  const APPLET_DETAILS = `${page.library}/${appletId}`;
+
+  const renderAppletInfo = () => {
+    switch (uiType) {
+      case AppletUiType.List:
```

Comment:
maybe we also need return in this case?

Reply:
there is the same return as for the next case